src/battle-dex.ts

```diff
@@ -852,6 +852,38 @@ const Tools = {
 		return type;
 	},
 
+	getAbilitiesFor: function (template: any, gen: number) {
```

Comment:
`getAbilitiesFor(`, not `getAbilitiesFor: function (`. Look at the surrounding code.

(Ditto `hasAbility`)